File: Java/src/main/java/com/intersystems/demo/pex/OutboundAdapter.java
package com.intersystems.demo.pex;

import com.intersystems.jdbc.IRISObject;

public class OutboundAdapter extends com.intersystems.enslib.pex.OutboundAdapter {
	
	public void OnInit() throws Exception {
		return;
	}

	public void OnTearDown() throws Exception {
		return;
	}

	public String printStringContainer(IRISObject stringContainer) {
		return "printed from java"+stringContainer;
	}

}

File: DotNet/OutboundAdapter.cs
using System;
using InterSystems.Data.IRISClient.ADO;

namespace InterSystems.Demo.PEX
{
    class OutboundAdapter : InterSystems.EnsLib.PEX.OutboundAdapter
    {
        public override void OnInit()
        {
            Console.Write("\r\n[.NET] ...Demo.PEX.OutboundAdapter:OnInit() is called");
            return;
        }

        public override void OnTearDown()
        {
            Console.Write("\r\n[.NET] ...Demo.PEX.OutboundAdapter:OnTearDown() is called");
            return;
        }

        public string printStringContainer(IRISObject stringContainer)
        {
            Console.WriteLine("\r\n[.NET] ...Demo.PEX.OutboundAdapter:printStringContainer(): " + (string)stringContainer.Get("StringValue"));
            return "printed successfully";
        }

    }
}
